db/migrate/20240522185716_create_good_job_process_lock_ids.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+class CreateGoodJobProcessLockIds < ActiveRecord::Migration[7.1]
+  def change
+    reversible do |dir|
+      dir.up do
+        # Ensure this incremental update migration is idempotent
+        # with monolithic install migration.
+        return if connection.column_exists?(:good_jobs, :locked_by_id)
+      end
+    end
+
+    change_table :good_jobs, bulk: true do |_t|
```

Comment:
strong migrations are unhappy about this, probably needed to wrap with `safety_assured` :thinking: